dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ReferenceConfig.java

```diff
@@ -241,7 +242,6 @@ public synchronized void destroy() {
             ModuleServiceRepository repository = getScopeModel().getServiceRepository();
             repository.unregisterConsumer(consumerModel);
         }
-        getScopeModel().getConfigManager().removeConfig(this);
```

Comment:
pls desc why remove this

Reply:
The first step of the destroy method will call super.destroy, which is `ReferenceConfigBase#destroy`, which has already done removeConfig